_includes/masthead.html

```diff
@@ -1,4 +1,4 @@
-<div class="masthead">
+<header role="banner" class="masthead">
```

Comment:
I think `role="banner"` is redundant. According to the W3 using `<header>` when its a descendant of the body automatically gets `role="banner"`.

> The HTML5 header element defines a banner landmark when its context is the body element.
>
> The HTML5 header element is not considered a banner landmark when it is descendant of any of following elements: `article`, `aside`, `main`, `nav`, `section`
>
> _ref: [Banner Landmark HTML5 techniques](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/banner.html)_